.travis.yml

```diff
@@ -31,7 +31,7 @@ script:
 
 env:
   matrix:
-    - PYTHONASYNCIODEBUG=1 AIOHTTP_NO_EXTENSIONS=0
-    - PYTHONASYNCIODEBUG=0 AIOHTTP_NO_EXTENSIONS=0
-    - PYTHONASYNCIODEBUG=1 AIOHTTP_NO_EXTENSIONS=1
-    - PYTHONASYNCIODEBUG=0 AIOHTTP_NO_EXTENSIONS=1
+    - PYTHONASYNCIODEBUG=nonemptystring AIOHTTP_NO_EXTENSIONS=
```

Comment:
Actually I prefer to explicitly state meaning of the environment variable to prevent confusions such as fixed in this PR.

Setting `PYTHONASYNCIODEBUG` to `yes`, `Y`, `1`,  `True`, `T` etc, most probably will make reader to think that to negate meaning of the variable he should use `no`, `N`, `0`,  `False`, `F`, etc value.

Maybe use shorter value, e.g. `set` instead of `nonemptystring`?